main.go

```diff
@@ -110,6 +115,7 @@ func convertRawDiskImageToQcow2(rawDiskUrl string) error {
 		"-r",
 		"curl",
 		rawDiskUrl,
+		"sslverify=false",
```

Comment:
This will be removed, as soon as I'll be able to use certificate that is provided by VMExport API and injected here (If I'm not mistaken).